addons/l10n_cn_standard/__manifest__.py

```diff
@@ -14,17 +14,21 @@
     'maintainer': 'jeff@osbzr.com',
     'website': 'http://openerp-china.org',
     'description': """
+包含企业会计准则以下数据
```

Comment:
Could you also add a description in English (can have both)? This way, non-speakers can understand what this module is about.